apps/src/javalab/sharedStyleConstants.js

```diff
@@ -0,0 +1,24 @@
+const headerSectionStyles = {
```

Comment:
Just a thought: since this seems to be the first of some common styling for Javalab, how do you feel about doing this in CSS instead?

Edit: oh looks like we already have a [javalab/style.scss](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/style/javalab/style.scss), but I'm not sure if we've been using that much?

Reply:
mmm good idea, let me give that a shot.

Reply:
@sanchitmalhotra126 tried to refactor into SCSS, take a look at this commit when you have a moment:

https://github.com/code-dot-org/code-dot-org/pull/44231/commits/2861c7b1504da34e290c26a89008d9ff433bfa8c